internal/clusterconfig/clusterconfig.go

```diff
@@ -49,14 +48,11 @@ const (
 	caBundleFileName = caBundleCMName + ".json"
 
 	networkDir = "/opt/openshift/network-configuration"
+
+	sshKeyFile = "/home/core/.ssh/authorized_keys.d/ignition"
```

Comment:
Consider adding a comment saying that this is the file that MCD/ignition writes the public ssh keys to